README.md

```diff
@@ -438,7 +438,7 @@ We have 4 options:
 1. Google - If you are building an "enterprise" or "education" app
 which you know will be used in Google-enabled companies/schools we
 wrote a Hapi plugin: https://github.com/dwyl/hapi-auth-google which
-lets you include Google Login in your app in a few simple steps. The plugin uses the [***Official Google Node.js API Client***](https://github.com/google/google-api-nodejs-client) and is
+lets you include Google Login in your app in a few steps. The plugin uses the [***Official Google Node.js API Client***](https://github.com/google/google-api-nodejs-client) and is
```

Comment:
How about in a few 'clear' steps? What we want to convey here is that an effort has been made to make the process as accessible as possible independently of a person's coding experience, so I feel like we need _some_ form of adjective here... Though I might just be being picky.